src/models/maps/bravia/defaultKeys.ts

```diff
@@ -0,0 +1,344 @@
+
+import { IElementConfig } from '../../interfaces';
+
+/**
+ * This is the list of most common commands from the Android TV Remote integration page.
```

Comment:
Change this Android TV specific docstring to instead point to here: https://www.home-assistant.io/integrations/braviatv/#remote